guides/gatherers.md

```diff
@@ -206,13 +206,45 @@ For extra information refer to [trento-project/agent/../gatherers/packageversion
 
 ### saphostctrl
 
-This gatherer returns allows access to certain webmethods that `saphostctrl` implements. At this moment only `Ping` and `ListInstances` are supported.
+This gatherer allows access to certain webmethods that `saphostctrl` implements. An argument is required to specify
+which webmethod should be called. This webmethod is passed to the `saphostctrl` command-line tool thought the `-function` argument. Note that not all webmethods are supported.
```

Comment:
I would personally remove the `Note that not all..` phrase. Having the list of supported is enough.